src/commands/dev/lib/dev-server.ts

```diff
@@ -420,7 +499,7 @@ export default class DevServer {
     nowRequestId: string,
     nowJson: NowConfig
   ) => {
-    const files = await this.getProjectFiles();
+    const files = this.files;
```

Comment:
```suggestion
    const { files } = this;
```

Reply:
There wasn't really a reason to have this variable. I updated the places it was used to use `this.files` instead.